protos/common.proto

```diff
@@ -18,6 +18,14 @@ enum AirbaseCategory {
   SHIP = 2;
 }
 
+enum GroupCategory {
+  GROUP_AIRPLANE = 0;
+  GROUP_HELICOPTER = 1;
+  GROUP_GROUND = 2;
+  GROUP_SHIP = 3;
+  GROUP_TRAIN = 4;
```

Comment:
Prefix is apparently necessary if multiple enums have the same variant, see the error I am getting otherwise:

```
\"SHIP\" is already defined in \"dcs\".\r\ncommon.proto:25:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  
Therefore, \"SHIP\" must be unique within \"dcs\", not just within \"GroupCategory\"
```

Reply:
I think that is fine, we may want to break down the gRPC namespace later but that is a problem for future, if it even becomes needed.

Reply:
I think this change is fine, we may want to break down the gRPC namespace later to avoid this but that is a problem for future (If it is even a real problem).

I am not familiar enough with gRPC in rust, C# and ruby (the languages currently in use) to evaluate the pros and cons so lets go with the above since it is the simplest approach.